packages/apps/src/SideBar/Item.tsx

```diff
@@ -15,27 +15,61 @@ import accountObservable from '@polkadot/ui-keyring/observable/accounts';
 import { withApi, withMulti, withObservable } from '@polkadot/ui-api/index';
 import { isFunction } from '@polkadot/util';
 
+import ReactTooltip from 'react-tooltip';
+
 type Props = I18nProps & ApiProps & {
+  isCollapsed: boolean,
   allAccounts?: SubjectInfo,
   route: Route
 };
 
+interface Tooltip {
+  'data-tip': boolean;
+  'data-for': string;
+  'data-tip-disable'?: boolean;
+}
+
 class Item extends React.PureComponent<Props> {
+
+  componentWillUpdate () {
+    ReactTooltip.rebuild();
+  }
```

Comment:
(nitpick) extra blank line after. (And the one before the method can be removed)